enterprise/workers/step-resolver/README.md

```
@@ -133,38 +133,112 @@ STEP_RESOLVER_HMAC_SECRET=local-dev-secret
+
+### One-time setup
+
+1. Create dispatch namespaces:
 
 ```bash
 pnpm run namespace:create:staging
 pnpm run namespace:create:production
+pnpm run namespace:create:production-eu
+```
+
+2. Add the `eu.step-resolver.novu.co` custom domain in the Cloudflare dashboard (same account as US).
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)